Documentation/ContentObjects/Pageview/Index.rst

```diff
@@ -0,0 +1,435 @@
+               <f:then>
+                  <!-- Item has children -->
+                  <a class="nav-link dropdown-toggle" data-bs-toggle="dropdown" href="#" role="button" aria-expanded="false">
+                     {menuItem.title}
+                  </a>
+                  <div class="dropdown-menu">
+                     <f:for each="{menuItem.children}" as="menuItemLevel2">
+                          <f:link.page pageUid="{menuItemLevel2.data.uid}"
+                                       class="dropdown-item {f:if(condition: menuItemLevel2.active, then: 'active')}">
+                             {menuItemLevel2.title}
+                          </f:link.page>
+                     </f:for>
+                  </div>
+               </f:then>
+               <f:else>
+                  <!-- Item has no children -->
+                  <f:link.page pageUid="{menuItem.data.uid}" class="nav-link {f:if(condition: menuItem.active, then:'active')}">
+                     {menuItem.title}
+                  </f:link.page>
+               </f:else>
+            </f:if>
+         </li>
+      </f:for>
+   </ul>
+
+..  _cobj-pageview-paths:
+
+paths
+-----
+
+..  confval:: paths.[priority]
+    :name: pageview-paths
+    :type: :ref:`data-type-path` with :ref:`stdWrap <stdwrap>`
+
+    Sets an array of paths for the Fluid templates, usually
+    :file:`EXT:my_extension/Resources/Private/Templates/` or a folder below that
+    path like :file:`EXT:my_extension/Resources/Private/Templates/MyPages`.
+
+    The templates are expected in a subfolder :path:`Pages` or
+    :path:`pages`.
+
+    Fluid partials are searched for in a subfolder called :path:`Partials` or
+    :path:`partials`, layouts in :path:`Layouts` or :path:`layouts`.
+
+    The name of the used page layout (:ref:`Backend layout <t3coreapi:be-layout>`)
+    is resolved automatically.
+
+    The paths are considered from the largest to the smallest priority.
+
+
+..  _cobj-pageview-paths-example:
+
+Example: Define a path that contains all templates
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This is a basic definition of a `PAGEVIEW` object with only one path to the
+templates:
+
+..  code-block:: typoscript
+    :caption: EXT:my_sitepackage/Configuration/TypoScript/setup.typoscript
+
+    page = PAGE
+    page.10 = PAGEVIEW
+    page.10.paths.100 = EXT:my_sitepackage/Resources/Private/Templates/
+
+The content of the folder could look like this:
+
+*   EXT:my_sitepackage/Resources/Private/Templates/
+
+    *   Layouts
+
+        *   Default.html
+        *   WithoutHeader.html
+
+    *   Pages
+
+        *   Default.html
+        *   StartPage.html
+        *   TwoColumns.html
+        *   With_sidebar.html
+
+    *   Partials
+
+        *   Footer.html
+        *   Sidebar.html
+        *   Menu.html
+
+If a page has a backend layout named "with_sidebar", the template file is
+then resolved
+to :file:`EXT:my_sitepackage/Resources/Private/Templates/Pages/With_sidebar.html`.
```

Comment:
Oh, didn't catch that, thought it would resolve to CamelCase. Odd.

Reply:
It was mentioned in the changelog